test.e2e/spec/services.spec.js

```diff
@@ -181,6 +181,89 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
             }
           );
         });
+
+        it('intercepts internal requests with internal urlBase',function() {
```

Comment:
I find the terms `intercepts` and `does not intercept` difficult to understand when reading a test name. Perhaps `sends auth credentials` and `does not send auth credentials` would work better?